src/vat/remotes.js

```diff
@@ -356,7 +409,10 @@ export function makeRemoteManager(
   }
 
   function sendTo(vatID, body) {
-    if (typeof body !== 'object' || !body.hasOwnProperty('op')) {
+    if (
+      typeof body !== 'object' ||
+      !Object.prototype.hasOwnProperty.call(body, 'op')
+    ) {
```

Comment:
let's add another TODO here, "can we just use 'op' in body". Everywhere we call sendTo we give it a brand new object, so I don't remember why we're guarding against inherited "op" properties. That verbose check is.. verbose.